app/views/directives/from-file.html

```diff
@@ -26,8 +26,6 @@
         }" ng-model="editorContent" class="editor ace-bordered yaml-mode" id="add-component-editor" required></div>
       </div>
 
-      <alerts alerts="alerts"></alerts>
```

Comment:
Removing this alert cause we alreadty have `alerts` directive in view https://github.com/openshift/origin-web-console/blob/master/app/views/create.html#L13